tests/test_validation/test_validation.py

```diff
@@ -0,0 +1,112 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+
+class Test_Validation(APITestCase):
+
+    def setUp(self):
```

Comment:
@nyabongoedgar , Please create a base test file and add these setUp methods.

Reply:
@nyabongoedgar I'm waiting on you to work on@Stanley-Okwii's feedback so that I can retest the review app on Heroku and approve your PR.

Reply:
@nyabongoedgar Please do remove that underscore from the class name.